egs/hi_mia/v1/RESULT

```diff
@@ -0,0 +1,6 @@
+run_kws_kf.sh
+shreld0.5
+accuracy 0.83273214201
```

Comment:
Could you please provide false alarm per hour and  recall?
These two metrics are more important in actual applications.